security/permissions/src/main/java/io/quarkus/ts/openshift/security/basic/permissions/beanparam/permissions/PossessedPermission.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.ts.openshift.security.basic.permissions.beanparam.permissions;
+
+import java.security.BasicPermission;
+import java.security.Permission;
+
+public class PossessedPermission extends BasicPermission {
```

Comment:
I'm confused by this naming. 

What is `Possessed` supposed to imply?

Reply:
Yes, it could be confused, it refers to the permission that the authenticated user has or possesses...
Quarkus checks if the permission the user possesses is enough "to imply" the permission required

Reply:
ok, so it is "acquired" permission